go.mod

```diff
@@ -53,6 +53,7 @@ require (
 	github.com/go-webauthn/webauthn v0.8.6
 	github.com/gobwas/glob v0.2.3
 	github.com/gogs/chardet v0.0.0-20211120154057-b7413eaefb8f
+	github.com/gogs/cron v0.0.0-20171120032916-9f6c956d3e14
```

Comment:
IIRC `gogs/cron` had been removed, now it is back?

Reply:
Yes, this is a block change. It needs to be changed to the new cron package or you could write the parser yourself. @appleboy